src/addrman_impl.h

```diff
@@ -0,0 +1,287 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_ADDRMAN_IMPL_H
+#define BITCOIN_ADDRMAN_IMPL_H
+
+#include <logging.h>
+
+#include <sync.h>
+#include <unordered_map>
+#include <unordered_set>
```

Comment:
Also include:

- cstdint (for fixed width integer types)
- netaddress.h (for CNetAddr)
- serialize.h (for serialization functions)
- protocol.h (for CAddress)
- uint256.h (for uint256)
- vector (for std::vector)
- utility (for std::pair)
- set (for std::set)
- optional (for std::optional)

Reply:
done